### ISSUE TITLE
submit annotation module for DHTech code review

### ISSUE THREAD

**Issue by kmcdono2** [Living-with-machines/MapReader#39]
The DHTech ADHO SIG is testing out a code review process for DH projects. We are considering submitting part of the MapReader code.

@kasra-hosseini suggested `mapImages`, annotation tool, or generally the install setup as options for a partial code review.

Submitting the code review requires preparing answers to questions documented [here](https://github.com/DHCodeReview/DHCodeReview/wiki/Authors:-Preparing-a-Code-Review). 

Tasks
- [ ] confirm which part of code to review
- [ ] prepare answers for submission
- [ ] make any updates to repo in preparation for submission (TBD)

**Comment by kmcdono2** [comment]
@rwood-97 @andrewphilipsmith Pinging you here so we could think about reviving the idea of sending out part of the MapReader code for review this spring with https://dhcodereview.github.io/. @kallewesterling is involved in this as well I think :)

**Comment by kallewesterling** [comment]
@kmcdono2 Is this something that you're still looking to do as a potential output of the project? Might be worth looking into -- now that we have the text detection as well?

**Comment by kmcdono2** [comment]
Yes, we were holding off since we just did the JOSS paper. I think it would be a good thing to do this w/text as you say.
